lib/agent0/agent0/utilities/docker.py

```diff
@@ -0,0 +1,60 @@
+"""Utilities associated to manipulating Docker."""
+import subprocess
+import logging
+import os
+import time
+from pathlib import Path
+
+
+def check_docker(restart: bool = False) -> None:
+    """Check whether docker is running to your liking.
+
+    Parameters
+    ----------
+    restart: bool
+        Restart docker even if it is running.
+    """
+    home_infra = Path(os.path.expanduser("~")) / "code" / "infra"
+    if os.path.exists(home_infra):
+        infra_folder = home_infra
+    else:
+        infra_folder = Path("/code/infra")
```

Comment:
this should not be hard-coding the `/`, right?

Reply:
removed in favor of passing in optional local path parameter in ea27a0edcbb15a0c5d5f6b03abc8d562274985bc